### PR TITLE
Bootstrap 5 + Popperjs 2

### PR DESCRIPTION
**Pull request by SaeedDev94**
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
- <https://getbootstrap.com/docs/5.0/components/dropdowns/#options>
- <https://popper.js.org/docs/v2/constructors/#options>

### CODE REVIEW

File: types/bootstrap/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "popper.js": "^1.14.1"
+        "popper.js": "^1.14.1",
```
Comment (martin-badin) [inline types/bootstrap/package.json:4]: Do you need the dependency?

Comment (Saeed-Pooyanfar) [reply types/bootstrap/package.json:4]: yes: `@popperjs/core`

Comment (martin-badin) [reply types/bootstrap/package.json:4]: and `popper.js` too?

Comment (Saeed-Pooyanfar) [reply types/bootstrap/package.json:4]: `popper.js` for bootstrap v4

**Review by martin-badin — APPROVED** [review]
Look good

**Review by martin-badin — CHANGES_REQUESTED** [review]
fix CI

**Comment by Saeed-Pooyanfar** [comment]
> fix CI

I made pull request to add `@popperjs/core`
https://github.com/microsoft/DefinitelyTyped-tools/pull/190

**Review by martin-badin — APPROVED** [review]

**Comment by Saeed-Pooyanfar** [comment]
Ready to merge

**Pull request merged**